### PR TITLE
has[ws] == false

### PR DESCRIPTION
**Pull request by samgermain**

### CODE REVIEW

File: ts/src/bigone.ts [hunk 1]
```diff
@@ -58,6 +58,7 @@ export default class bigone extends Exchange {
                 'fetchWithdrawals': true,
                 'transfer': true,
                 'withdraw': true,
+                'ws': undefined,  // has but not implemented
```
[the remark this answers is not in the document]
Comment (samgermain) [reply ts/src/bigone.ts:61]: Yeah it's only really there to help us

[the remark this answers is not in the document]
Comment (samgermain) [reply ts/src/bigone.ts:61]: @carlosmiei I can remove all the instances of `                'ws': undefined,  // has but not implemented` if you want, I just thought it made it easier to check which exchanges still needed it

**Comment by samgermain** [comment]
@carlosmiei Any more issues?

**Comment by carlosmiei** [comment]
LGTM

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**